packages/build-info/src/file-system.test.ts

```diff
@@ -85,14 +86,28 @@ describe.concurrent('Test the platform independent base functionality', () => {
   })
 
   test('join should join path segments and always replace to posix style', ({ fs }) => {
+    expect(fs.join('/', 'package.json')).toBe('/package.json')
+    expect(posixJoin('/', 'package.json')).toBe('/package.json')
```

Comment:
added the `posixJoin` as baseline as this is the official implementation of posix join.